core/src/main/java/eu/xenit/alfresco/AntiIdleScheduledJobExecuter.java

```diff
@@ -1,11 +1,38 @@
 package eu.xenit.alfresco;
 
+import static org.alfresco.service.namespace.QName.createQName;
+
+import java.io.Serializable;
+import java.time.LocalDateTime;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import org.alfresco.model.ContentModel;
+import org.alfresco.repo.nodelocator.CompanyHomeNodeLocator;
+import org.alfresco.repo.nodelocator.NodeLocatorService;
+import org.alfresco.repo.transaction.RetryingTransactionHelper;
 import org.alfresco.service.ServiceRegistry;
+import org.alfresco.service.cmr.repository.ChildAssociationRef;
+import org.alfresco.service.cmr.repository.NodeRef;
+import org.alfresco.service.cmr.repository.NodeService;
+import org.alfresco.service.namespace.NamespaceService;
+import org.alfresco.service.namespace.QName;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import org.springframework.util.StringUtils;
 
 public class AntiIdleScheduledJobExecuter {
     private static final Logger LOG = LoggerFactory.getLogger(AntiIdleScheduledJobExecuter.class);
+    private static final String defaultLocation = "/Company Home/Data Dictionary/Anti Idle/Dummy Document";
```

Comment:
I have a tingling feeling about this default: are we sure "Data Dictionary" can't be translated if you start Alfresco with a different locale ?

Reply:
Good question, I will try it out

Reply:
You can side-step that by using qname-path ?
Or can/should we just hide this object somewhere outside the "Company Home" tree ?

Reply:
I've hidden it outside of the Company Home tree